hlapipeline/alignimages.py

```diff
@@ -231,19 +231,30 @@ def perform_align(input_list, archive=False, clobber=False, update_hdr_wcs=False
 
     for imgname in extracted_sources.keys():
         table=extracted_sources[imgname]["catalog_table"]
+
+        # Get the location of the current image in the filtered table
+        index = np.where(filteredTable['imageName']==imgname)[0][0]
+
+        # First ensure sources were found
+        if table[1] == None:
```

Comment:
Is there a reason you are evaluating the second element in the table ('table[1]'), and not the first ('table[0]')?

Reply:
Since my Python is rudimentary, I test in an interactive session.  I performed the same steps as the real code on the troublesome dataset.   Doing table[0] claims "KeyError: 0" where table[1] has None.

Reply:
If I print(table), I get {1"None}.